objects/9ad187/script.lua

```diff
@@ -42,11 +42,13 @@ optionalBlightSetup = true
 optionalExtraBoard = false
 optionalThematicRedo = false
 optionalBoardPairings = true
+optionalScaleBoard = false
 
 exploratoryVOTD = false
 exploratoryBODAN = false
 exploratoryWar = false
 
+
```

Comment:
what's this extra line for?

Reply:
My guess is an extra line from when the variable was named differently. Removed.